app/controllers/api/v1/merchants/search_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::Merchants::SearchController < ApplicationController
+  def show
+    render json: MerchantSerializer.new(find_merchants(params).first)
+  end
+
+  # def index
```

Comment:
This action is not working how I have envisioned it. It's passing test but my json hash is null? Will have to work on find_all